### PR TITLE
FDTN-301: Enabling JVM metrics in DD

### PR DESCRIPTION
**Pull request by Yatharth0045**

### CODE REVIEW

File: api/conf/base.conf [hunk 1]
```diff
@@ -30,8 +30,25 @@ rollbar.token = ${?ROLLBAR_TOKEN}
 
 JWT_SALT = ${?JWT_SALT}
 
+cinnamon.datadog {
+  statsd {
+    host = "datadog-cluster-agent.datadog"
```
Comment (gheine) [inline api/conf/base.conf:35]: This should come from the environment, if available. Something like
host = "localhost"
host = ${?DD_AGENT_HOST}

File: api/conf/base.conf [hunk 2]
```diff
@@ -30,8 +30,25 @@ rollbar.token = ${?ROLLBAR_TOKEN}
...
+    port = 5005
```
Comment (gheine) [inline api/conf/base.conf:36]: Port should probably not be set and instead use the default

File: api/conf/base.conf [hunk 3]
```diff
@@ -30,8 +30,25 @@ rollbar.token = ${?ROLLBAR_TOKEN}
...
+  reporters += statsd-reporter
+
+  statsd-reporter {
+
+    dogstatsd {
+      enabled = on
+    }
+  }
+
   datadog-reporter = ${cinnamon.chmetrics.statsd-reporter} {
```
Comment (gheine) [inline api/conf/base.conf:52]: this syntax looks weird. Can you link to the documentation please?

**Review by gheine — CHANGES_REQUESTED** [review]
Some comments inline

File: api/conf/base.conf (hunk 3 above)
Comment (Yatharth0045) [reply api/conf/base.conf:52]: This is for the newer version. We can ignore this for now. @gheine

Comment (Yatharth0045) [reply api/conf/base.conf:52]: But here's the reference to the old version documentation
https://developer.lightbend.com/docs/telemetry/2.3.x/plugins/datadog/datadog.html
https://developer.lightbend.com/docs/telemetry/2.3.x/plugins/chmetrics/statsd.html#dogstatsd

File: api/conf/base.conf (hunk 1 above)
Comment (Yatharth0045) [reply api/conf/base.conf:35]: Will do it after testing if this static conf works as expected

**Pull request merged**